DXMainClient/DXGUI/Multiplayer/LANGameLoadingLobby.cs

```diff
@@ -667,5 +672,21 @@ protected override void HandleGameProcessExited()
 
             LeaveGame();
         }
+
+        protected override void UpdateDiscordPresence(bool resetTimer = false)
+        {
+            if (discordHandler == null)
+                return;
+
+            PlayerInfo player = Players.Find(p => p.Name == ProgramConstants.PLAYERNAME);
+            if (player == null || lblMapNameValue == null || lblGameModeValue == null)
```

Comment:
`lblMapNameValue` and `lblGameModeValue` are UI controls, they can never be null.

Reply:
Replaced null checks with `Text` property null-or-empty checks.